server/controllers/votesController.js

```diff
@@ -0,0 +1,82 @@
+import 'babel-core/register';
+import 'babel-polyfill';
+import db from '../models/index';
+
+
+/**
+ *
+ *
+ * @export
+ * @class Vote
+ */
+export default class Vote {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} with a successful message and recipe with the vote
+   * @memberof Vote
+   */
+  static async voteRecipe(req, res) {
+    const voteQuery = req.query.vote;
+    if (!voteQuery) {
+      return res.status(400).json({ message: 'Action is required.' });
+    }
+
+    const { userId } = req;
```

Comment:
You can combine your destructuring like so:
```
const { userId, params: { recipeId } } = req;
```

Reply:
Fixed

Reply:
Fixed